R/check.R

```diff
@@ -91,19 +99,22 @@ check_module <- function(module) {
 #' @return NULL
 #'
 #' @family check
+#' @importFrom rlang warn
 check_modules_compatible <- function(modules) {
   if (
     modules[[4]]$data$model_miss &&
       !modules[[6]]$data$likelihood_aggregation
   ) {
-    warning(
-      "Incompatible model specification: A missingness model has ",
-      "been specified but likelihood aggregation is specified as ",
-      "by snapshot. Switching to likelihood aggregation by group.",
-      " This has no effect on the nowcast but limits the ",
-      "number of threads per chain to the number of groups. To ",
-      "silence this warning, set the `likelihood_aggregation` ",
-      "argument in `enw_fit_opts` to 'groups'.",
+    rlang::warn(
+      paste0(
+        "Incompatible model specification: A missingness model has ",
+        "been specified but likelihood aggregation is specified as ",
+        "by snapshot. Switching to likelihood aggregation by group.",
+        " This has no effect on the nowcast but limits the ",
+        "number of threads per chain to the number of groups. To ",
```

Comment:
Prefer to line break on sentences, rather than maximize characters used.

Reply:
if this is for another issue can it turned into one and this resolved?